File: app/src/main/java/guillermo/com/pugme/ui/activities/PugsActivity.java
package guillermo.com.pugme.ui.activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import butterknife.Bind;
import butterknife.ButterKnife;
import guillermo.com.pugme.R;
import guillermo.com.pugme.ui.fragments.PugsGridFragment;

public class PugsActivity extends AppCompatActivity {

    @Bind(R.id.toolbar)
    Toolbar toolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pugs);
        ButterKnife.bind(this);
        toolbar.setTitle("");
        setSupportActionBar(toolbar);

        getSupportFragmentManager().beginTransaction()
                .add(R.id.pugs_fragment_container, new PugsGridFragment()).commit();

    }


}

File: app/src/main/java/guillermo/com/pugme/ui/activities/PugsActivity.kt
package guillermo.com.pugme.ui.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import butterknife.ButterKnife
import guillermo.com.pugme.R
import guillermo.com.pugme.ui.fragments.PugsGridFragment
import kotlinx.android.synthetic.main.toolbar.*

class PugsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pugs)
        ButterKnife.bind(this)
        toolbar?.title = ""
        setSupportActionBar(toolbar)

        supportFragmentManager.beginTransaction().add(R.id.pugs_fragment_container, PugsGridFragment()).commit()

    }


}
